sky/shell/platform/android/vk/vulkan_proc_table.h

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SKY_SHELL_PLATFORM_ANDROID_VK_VULKAN_PROC_TABLE_H_
+#define FLUTTER_SKY_SHELL_PLATFORM_ANDROID_VK_VULKAN_PROC_TABLE_H_
+
+#include "lib/ftl/macros.h"
+#include "vulkan_interface.h"
+
+namespace sky {
+namespace shell {
+
+class VulkanProcTable {
```

Comment:
Yeah, I didn't know where to put it so just picked a spot. But the directory has its own GN file and is otherwise relocatable. Will move to //flutter/vulkan and update the headers guards and such.